mirage/serializers/customer-resource.js

```diff
@@ -1,30 +1,28 @@
 import ApplicationSerializer from './application';
 
-export default ApplicationSerializer.extend({
-  include: ['package', 'title', 'visibilityData', 'customEmbargoPeriod', 'managedEmbargoPeriod', 'managedCoverageList'],
-
-  serialize(...args) {
-    let json = ApplicationSerializer.prototype.serialize.apply(this, args);
+function mapCustomerResourceAttrs(hash, customerResource) {
+  hash.attributes.vendorId = customerResource.package.vendor.id;
+  hash.attributes.vendorName = customerResource.package.vendor.name;
+  hash.attributes.packageId = customerResource.package.id;
+  hash.attributes.packageName = customerResource.package.name;
+  hash.attributes.contentType = customerResource.package.contentType;
+  hash.attributes.titleId = customerResource.title.id;
+  hash.attributes.name = customerResource.title.name;
+  hash.attributes.publisherName = customerResource.title.publisherName;
+  hash.attributes.publicationType = customerResource.title.publicationType;
+  hash.attributes.contributors = customerResource.title.contributors;
+  hash.attributes.identifiers = customerResource.title.identifiers;
+  return hash;
```

Comment:
this may no longer be necessary with the new customerResource objects being flattened

Reply:
This is the flattening for mirage. Otherwise we'd have to define all of those properties twice and still make sure the customer-resource attributes equal the corresponding title attributes.